urllib3/util/ssl_.py

```diff
@@ -195,6 +197,16 @@ def create_urllib3_context(ssl_version=None, cert_reqs=ssl.CERT_REQUIRED,
     """
     context = SSLContext(ssl_version or ssl.PROTOCOL_SSLv23)
 
+    if hasattr(context, 'urllib3_wrapper_context'):
```

Comment:
I'm not sure what you mean by test file in this context. You can restore the ``SSLContext`` after the test has been ran, however since this is going to do different things at the module level based on an import error you're also going to need to delete this module from ``sys.modules`` to trigger another import. I'm not sure if it's worth the machinations required to test that.

Another reason to do the warning there, is if this API goes away and urllib3 starts using the ``SSLContext`` directly then this warning has less likely to get lost in the shuffle.